scrapers.js

```diff
@@ -253,17 +256,18 @@ let scrapers = [
       let latestDate = data[data.length - 1].data;
 
       // Get only records for that date
-      return data.filter((row) => {
-        return row.data === latestDate;
-      })
-      .map((row) => {
-        return {
-          recovered: parse.number(row.dimessi_guariti),
-          deaths: parse.number(row.deceduti),
-          cases: parse.number(row.totale_casi),
-          county: parse.string(row.denominazione_regione)
-        };
-      });
+      return data
+        .filter(row => {
+          return row.data === latestDate;
+        })
+        .map(row => {
+          return {
+            recovered: parse.number(row.dimessi_guariti),
+            deaths: parse.number(row.deceduti),
+            cases: parse.number(row.totale_casi),
+            county: parse.string(row.denominazione_regione)
+          };
+        });
```

Comment:
Erm, I'm confused, why is Italy modified here, and why does the existing code not have `row.data.substr(0, 10)`? Can you rebase from master?